runtime/src/iree/hal/drivers/vulkan/registration/driver_module.cc

```diff
@@ -42,7 +42,7 @@ IREE_FLAG(bool, vulkan_sparse_residency, true,
 IREE_FLAG(
     bool, vulkan_dedicated_compute_queue, false,
     "Use a dedicated queue with VK_QUEUE_COMPUTE_BIT for dispatch workloads.");
-IREE_FLAG(bool, vulkan_vma_allocator, true,
+IREE_FLAG(bool, vulkan_vma_allocator, false,
```

Comment:
Interesting test failures... I wonder if they reproduce consistently.
https://github.com/openxla/iree/actions/runs/5927315254/job/16070945531?pr=14760#step:7:833
`/work/runtime/src/iree/hal/drivers/vulkan/native_allocator.cc:300: RESOURCE_EXHAUSTED; VK_ERROR_OUT_OF_DEVICE_MEMORY; vkAllocateMemory; failed to allocate buffer of length 24; while invoking native function hal.allocator.allocate; while calling import; `

Reply:
yeah very weird - building now to try to reproduce locally